readme.md

```diff
@@ -51,6 +46,7 @@ https://www.youtube.com/watch?v=yz1PsuLLpho
 
 ### Help Wanted
 >
+> * Would love someone to convert from FireSpitter to Stock and/or KSPWheel  
```

Comment:
there isn't any parts that benefit from FireSpitter - this was accidentally left in when copied from template.